source/Assets/Plugins/Didomi/Scripts/Android/AndroidObjectMapper.cs

```diff
@@ -204,6 +204,23 @@ public static IList<string> GetMethodListString(AndroidJavaObject obj, string me
             return null;
         }
 
+        public static DidomiInitializeParameters ConvertToDidomiInitializeParameters(AndroidJavaObject obj)
+        {
+            var initializeParameters = new DidomiInitializeParameters(
```

Comment:
Is it to convert parameters from Java / Kotlin to C#? I am not sure why we would need it.

Reply:
you're right, i just removed the function